src/Network/Wai/Handler/Hal.hs

```diff
@@ -31,6 +31,8 @@
 -- @
 module Network.Wai.Handler.Hal
   ( run,
+    Options (..),
+    defaultOptions,
     runWithContext,
```

Comment:
I would prefer if we used the following order in the export list and when declaring these types and functions within the module:
```suggestion
    runWithContext,
    Options (..),
    defaultOptions,
```
Reason: The reader is then presented with the two main entry points to the module, and then the "what's an `Options`, and how do I make one?" is immediately raised and answered.